datahub-web-react/src/app/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,693 @@
+                />
+                <Typography.Title level={3}>Welcome to DataHub! 👋</Typography.Title>
+                <Typography.Paragraph style={{ lineHeight: '22px' }}>
+                    <strong>DataHub</strong> helps you discover and organize the important data within your
+                    organization. You can:
+                </Typography.Paragraph>
+                <Typography.Paragraph style={{ lineHeight: '24px' }}>
+                    <ul>
+                        <li>
+                            Quickly <strong>search</strong> for Datasets, Dashboards, ML models, and more
+                        </li>
+                        <li>
+                            View and understand the full <strong>end-to-end Lineage</strong> of how data is created,
+                            transformed, and consumed
+                        </li>
+                        <li>
+                            Gain <strong>insights</strong> about how others within your organization are using data
+                        </li>
+                        <li>
+                            Define <strong>ownership</strong> and capture <strong>knowledge</strong> to empower others
+                        </li>
+                    </ul>
+                    <p>Let&apos;s get started! 🚀</p>
+                    <div
+                        style={{
+                            backgroundColor: ANTD_GRAY[4],
+                            opacity: '0.7',
+                            borderRadius: '4px',
+                            height: '40px',
+                            display: 'flex',
+                            alignItems: 'center',
+                        }}
+                    >
+                        <span style={{ paddingLeft: '5px' }}>💡</span>
+                        <span style={{ paddingLeft: '10px' }}>
+                            Press <strong> Cmd + Ctrl + T</strong> to open up this tutorial at any time.
+                        </span>
+                    </div>
+                </Typography.Paragraph>
+            </div>
+        ),
+        style: { minWidth: '650px' },
+    },
+    {
+        id: HOME_PAGE_INGESTION_ID,
+        selector: `#${HOME_PAGE_INGESTION_ID}`,
+        title: 'Ingest Data',
+        content: (
+            <Typography.Paragraph>
+                Start integrating your data sources immediately by navigating to the <strong>Ingestion</strong> page.
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: HOME_PAGE_DOMAINS_ID,
+        selector: `#${HOME_PAGE_DOMAINS_ID}`,
+        title: 'Explore by Domain',
+        content: (
+            <Typography.Paragraph>
+                Here are your organization&apos;s <strong>Domains</strong>. Domains are collections of data assets -
+                such as Tables, Dashboards, and ML Models - that make it easy to discover information relevant to a
```

Comment:
Avoid mentioning ML Models - They are not very well supported in DataHub today. Replace with Data Pipelines?

Reply:
Done!